README.md

```diff
@@ -43,13 +43,17 @@ For that we need to do something like the `-S` flag before and I prefer to use `
 
 Because `yay` cover all the db (core, extra, aur, ...) and `pacman` handle only core and extra.
 
-## Manual installation
+## Installation
 
-Place the source (`a2n.archupdate.plasmoid` folder) in `~/.local/share/plasma/plasmoids/` or dl via [the KDE store](https://www.pling.com/p/1940819/)
+### Automatic installation
 
-## Automatic installation
+You can install the widget from the [AUR](https://aur.archlinux.org/packages/kdeplasma-arch-update-notifier-git) with `yay -S kdeplasma-arch-update-notifier-git`
 
-Dl via the pling store or via the `get new widgets...` window.
+Alternatively: download via the pling store or via KDEs `★ Get New Widgets...` window.
```

Comment:
The starts icon is set by your theme 
![2023-11-09_09-52](https://github.com/bouteillerAlan/archupdate/assets/30547569/7a7ae497-e2d4-4dd7-b69e-5aff6b875127)

I think we should write : 

> You can install the widget from : 
> - the KDE menu `Get New Widgets...`
> - from the [AUR](https://aur.archlinux.org/packages/kdeplasma-arch-update-notifier-git), eg. `yay -s kdeplasma-arch-update-notifier-git`

The dl from pling is not necessarily automatic, because you can choose between two way of downloading it.